scan-framework/src/main/java/com/getbouncer/scan/framework/time/Timer.kt

```diff
@@ -56,4 +60,23 @@ private class LoggingTimer(
         }
         return result
```

Comment:
you can probably replace the `measure` method with something like
```
override fun <T> measure(taskName: String?, task: () -> T): T =
    runBlocking { measureSuspend(taskName, task) }
```

Reply:
In fact, you can probably just make this a concrete implementation in the `Timer` class so that subclasses do not have to implement these separately.